cookbooks/HuggingFace/hf.py

```diff
@@ -153,7 +153,9 @@ def __init__(self, model_id: str = None, use_api_token=False):
         token = None
 
         if use_api_token:
-            token = maybe_get_api_key_from_environment("HUGGING_FACE_API_TOKEN")
+            # You are allowed to use Hugging Face for a bit before you get
+            # rate limited, in which case you will receive a clear error
+            token = maybe_get_api_key_from_environment("HUGGING_FACE_API_TOKEN", required=False)
```

Comment:
why is required=False?

Reply:
I left a comment, but it's because Hugging Face allows you to use their API for a bit before you get rate-limited. When you get rate limited, we get a very clear error from Hugging Face saying to login and use an API key